README.md

```diff
@@ -139,6 +139,96 @@ command groups. The `NullOrchestrator` disables commands related to managing ser
 orchestrator=NullOrchestrator()
 ```
 
+#### HelmOrchestrator
+
+The project also includes a [helm](https://helm.sh/docs/intro/quickstart/) orchestrator for deploying charts to [kubernetes](https://kubernetes.io/) clusters.
+
+Create a new `resources` directory as follows:
+
+```bash
+drwxrwxr-x  resources/
+.rw-rw-r-- ├──  settings.yml
+drwxrwxr-x └──  templates/
+drwxrwxr-x    ├──  baseline/
+drwxrwxr-x    │  └──  cli/
+drwxrwxr-x    │     └──  helm/
+drwxrwxr-x    │        ├──  values/
+.rw-rw-r--    │        │  └──   values.yml
+drwxrwxr-x    │        ├──  values-files/
+.rw-rw-r--    │        └──   mychart.tgz
+drwxrwxr-x    └──  configurable/
+drwxrwxr-x       └──  cli/
+drwxrwxr-x          └──  home/
+drwxrwxr-x             └──  .kube/
+.rw-------                └──  config  # Overwrite this with a cluster specific config file. ie `~/.kube/config`.
+```
+
+You can then define the orchestrator:
+
+```python
+from appcli.orchestrators import HelmOrchestrator
+orchestrator = HelmOrchestrator(
+    # Name of the project.
+    release_name="myapp",
+
+    # Chart archive path (relative to `conf/templates/cli/helm/`).
+    # [Optional] Default is `chart/`
+    chart_location="mychart.tgz",
+
+    # Chart source path (absolute).
+    # [Optional] Default is `None`.
+    dev_chart_location="path/to/mychart"
+)
+```
+
+_NOTE:_ Using `dev_chart_location` allows you to specify the location to your charts source code, so that live changes can be reflected every time `./myapp service start` is run.
```

Comment:
This needs to be clearer. Explain when/why you would use it.
Presumably it is ONLY for development purposes, the final version should never include it.
In which case, we're better off not having it as a config value here, and rather using a ENV var to handle it.

```
MYAPP_DEV_MODE=true MY_APP_DEV_MODE_HELM_CHART=/path/to/blah ./myapp.py
```